### PR TITLE
Migrate the embargo administration table to the new implementation

### PR DESCRIPTION
**Pull request by wtritch**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report-table.component.html [hunk 1]
```diff
@@ -46,7 +46,6 @@
                let-columns="columns">
     <tr>
       <td *ngFor="let column of columns; let columnIdx = index;"
-          [ngSwitch]="column.id"
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report-table.component.html]: Unrelated cleanup, noticed a double-switch in the aggregate report table

**Comment by wtritch** [comment]
Merging this now, will resolve any comments as they arrive.

File: webapp/src/main/webapp/src/app/admin/embargo/embargo-table.component.html [hunk 2]
```diff
@@ -1,82 +1,96 @@
...
+
+  <ng-template pTemplate="header"
+               let-columns>
+    <tr>
+      <th *ngFor="let column of columns"
+          [ngSwitch]="column.id">
+
+        <div *ngSwitchCase="'subject'"
+              class="text-right">
+          {{translateContext + '.headers.' + column.code | translate}}
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/embargo/embargo-table.component.html]: we should standardize our column interfaces probably i think i've seen us use code/headerKey etc.

Comment (wtritch) [reply webapp/src/main/webapp/src/app/admin/embargo/embargo-table.component.html]: I keep debating that, but they're also super-tightly coupled to their individual views... I'm not sure this is something I *want* to have a base-class or interface for...

Comment (esotrope) [reply webapp/src/main/webapp/src/app/admin/embargo/embargo-table.component.html]: i feel like - base class - _no_, established convention - _ideally_
